### ISSUE TITLE
Integrate StackOverflow Notifier

### ISSUE THREAD

**Issue by yukiisbored** [coala/corobo#46]

**Comment by Makman2** [comment]
So it notifies you if there's a coala related thread/question opened? That would be awesome :O

**Comment by meetmangukiya** [comment]
> So it notifies you if there's a coala related thread/question opened? That would be awesome :O

No, not really. This was meant as a gitter integration for stack overflow for selected tags.

But what you say is good as well.
